### PR TITLE
[TOMEE-4109] TomEE 8.x Update velocity to 2.3 and avoid CVE warnings

### PR DESCRIPTION
**Pull request by sultan**
Update dependency
* velocity 2.3

### CODE REVIEW

File: tomee/tomee-embedded/src/main/java/org/apache/tomee/embedded/Container.java [hunk 1]
```diff
@@ -1016,7 +1015,6 @@ private void copyTemplateTo(final File targetDir, final String filename) throws
 
         // don't break apps using Velocity facade
         final VelocityEngine engine = new VelocityEngine();
-        engine.setProperty(Velocity.RUNTIME_LOG_LOGSYSTEM, new NullLogChute());
```
Comment (sultan) [inline tomee/tomee-embedded/src/main/java/org/apache/tomee/embedded/Container.java:1019]: there is no replacement for LogChute

**Comment by rzo1** [comment]
Can you create a jira?

**Comment by sultan** [comment]
issue created on JIRA, waiting review before this can be merged

**Review by rzo1 — APPROVED** [review]

**Comment by sultan** [comment]
@rzo1, before merging, can you confirm it is ok to lose the LogChute features?

**Comment by rzo1** [comment]
There is no `LogChute` replacement. I am using 2.x for a long time with TomEE now w/o issues, so I guess it is ok to loose it.

**Comment by sultan** [comment]
Good to go

**Pull request merged**